src/libraries/apicompat/ApiCompat.proj

```diff
@@ -95,6 +95,36 @@
            AlwaysCreate="true" />
   </Target>
 
+  <Target Name="AddApiCompatResolvedCoreLibProjectReferences">
+    <ItemGroup>
+      <_apiCompatResolvedCoreLibProjectReference Include="@(_ResolvedProjectReferencePaths->WithMetadataValue('MSBuildSourceProjectFile','$(CoreLibProject)'))" />
+      <_apiCompatCoreLibReferenceAssembly Include="@(_apiCompatResolvedCoreLibProjectReference->Metadata('ReferenceAssembly'))" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="RunApiCompatForCoreLib"
```

Comment:
This projects compare the current live built assemblies against prebuilts (net6.0, netstandard2.1 and netstandard2.0). The ApiCompat comparison for CoreLib should just happen as part of the CoreLib build similar to how this works for every other library.